lib/sqlalchemy/orm/util.py

```diff
@@ -932,6 +932,10 @@ def with_polymorphic(
        only be specified if querying for one specific subtype only
     """
     primary_mapper = _class_to_mapper(base)
+
+    if selectable not in (None, False) and flat:
+        raise sa_exc.ArgumentError("Argument cannot be selectable and flat")
```

Comment:
"the 'flat' and 'selectable' arguments cannot be passed simultaneously to with_polymorphic()"